src/winmain.cpp

```diff
@@ -69,6 +72,98 @@ gup [-verbose] [-vVERSION_VALUE] [-pCUSTOM_PARAM]\r\
 
 std::string thirdDoUpdateDlgButtonLabel;
 
+class CUXHelper
+{
+public:
+	CUXHelper()
+	{
+		_hUXTheme = ::LoadLibrary(TEXT("uxtheme.dll"));

```

Comment:
Why do we need uxtheme ?

Reply:
It is needed for API `EnableThemeDialogTexture`. This piece of code was taken from Notepad++. 

https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/src/Parameters.cpp#L1126-L1132

Reply:
Yes, I know. It was for drawing tab interface dialog.
My question is, why do we need it here?

Reply:
See the difference with and without (specially dialog background)

![image](https://user-images.githubusercontent.com/14791461/140523949-7f5ddfce-46f4-44cc-956e-e0a0d4d43719.png)

Reply:
I do remember that I did add it for tab interface (Find dialog for example) so the dialog color could go with white tab. This usage has been removed after Windows 10 being popular. The reason to keep `EnableThemeDialogTexture` is for remaining backward-compatibility of plugins (surly some plugins still use `NPPM_GETENABLETHEMETEXTUREFUNC`).

So it's OK for me the case of left image (Without). As well, without `EnableThemeDialogTexture` part it's will meet the contribution guideline: `Single feature or bug-fix per PR.` (https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#guidelines-for-pull-requests).